rcfc/demo.py

```diff
@@ -21,14 +21,27 @@ def simple_button2():
     print("You pressed a different button!")
 
 
+@button.simple("Group Button!", group="Group A")
```

Comment:
I do like that I can make any button a group button, though. I started going down the road of making a separate type of button, a 'group' button, but realized that would mean you'd need a group slider, group toggle, group simple, etc. Seemed like too much. I think it makes sense to have it as an additional feature to the current buttons.

Reply:
I much prefer the way you have it set up.